functions/index.js

```diff
@@ -7,13 +7,22 @@
  * See a full list of supported triggers at https://firebase.google.com/docs/functions
  */
 
-const { onRequest } = require("firebase-functions/v2/https");
-const logger = require("firebase-functions/logger");
-
 // Create and deploy your first functions
 // https://firebase.google.com/docs/functions/get-started
 
-// exports.helloWorld = onRequest((request, response) => {
-//   logger.info("Hello logs!", {structuredData: true});
-//   response.send("Hello from Firebase!");
-// });
+const functions = require("firebase-functions");
+const express = require("express");
+const app = express();
+
+const admin = require("firebase-admin");
+const serviceAccount = "./service_account_key.json";
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount)
+});
+
+app.get("/", (req, res) => {
+  res.send("Welcome to ChatDev's Backend!");
+});
+exports.app = functions
+  .region("us-east1")
+  .https.onRequest(app);
```

Comment:
This is syntax for v1. I want you to use v2. If you look at the Firebase Functions page you will see that it says "v1"